source/quark/R/prepData.R

```diff
@@ -106,7 +112,11 @@ prepData <- function(rawData,
         cleanData(map = quarkData)
 
         ## Find any (bivariate) collinear variables:
-        findCollin(map = quarkData)
+        ## If not using Parallel process
+        #if(!useParallel)
+         # findCollin(map = quarkData)
+        #else ## If using Parallel process
+          findCollin(map = quarkData, f_useParallel = useParallel, f_nProc = nProcess)
```

Comment:
The "f_useParallel" and "f_nProc" arguments in this function call do not exist in the function definition.